mongoid.gemspec

```
@@ -38,7 +38,7 @@ Gem::Specification.new do |s|
   # Ruby 3.0 requires ActiveModel 6.0 or higher.
   # activemodel 7.0.0 cannot be used due to Class#descendants issue
   # See: https://github.com/rails/rails/pull/43951
-  s.add_dependency("activemodel", ['>=5.1', '<7.2', '!= 7.0.0'])
+  s.add_dependency("activemodel", ['>=5.1', '<8.0', '!= 7.0.0'])
```

```suggestion
  s.add_dependency("activemodel", ['>=5.1', '<7.3', '!= 7.0.0'])
```
Since Mongoid heavily depends on some ActiveModel internals, we should be careful with the upper limit.

---

But rails's next version is already 8.0...?
https://edgeguides.rubyonrails.org/8_0_release_notes.html

---

@comandeo-mongo as @PikachuEXE said the next version is 8.0 we don't have 7.3 but if you want ok :)